<launch>
    <node name="dummy_node" pkg="rosen_abstract_node" type="dummy_node" output="screen" />
</launch>
